aos/closed_loop/ClosedLoopSimulation.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>I would like to see %%time at the beginning of each cell - that gives you a general idea of how long it took to run the cell, which is useful for benchmarking</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Implemented</p>